app/src/main/java/com/concordium/wallet/ui/recipient/scanqr/ScanQRActivity.kt

```diff
@@ -222,6 +222,8 @@ class ScanQRActivity : BaseActivity(R.layout.activity_scan_qr, R.string.scan_qr_
         permissions: Array<String?>,
         grantResults: IntArray
     ) {
+        super.onRequestPermissionsResult(requestCode, permissions, grantResults)
+        super.onRequestPermissionsResult(requestCode, permissions, grantResults)
```

Comment:
Why is this line duplicated?

Reply:
yes, this should be single line. I'll remove it.